src/main/java/ca/serum390/godzilla/api/handlers/GoodsHandler.java

```diff
@@ -21,11 +21,9 @@
 public class GoodsHandler {
```

Comment:
Try to avoid autoformatting unless the code needs it. This file was already well formatted, and the autoformatter has messed it up to where it is less readable.